ITR/data/base_providers.py

```diff
@@ -358,44 +392,63 @@ def get_company_intensity_and_production_at_base_year(self, company_ids: List[st
                           self.column_config.GHG_SCOPE12,
                           self.column_config.GHG_SCOPE3]]
         ei_at_base = self._get_company_intensity_at_year(base_year, company_ids).rename(self.column_config.BASE_EI)
-        return company_info.merge(ei_at_base, left_index=True, right_index=True)
+        df = company_info.merge(ei_at_base, left_index=True, right_index=True)
+        df.reset_index('scope', inplace=True)
+        cols = df.columns.tolist()        
+        df = df[cols[1:3] + [cols[0]] + cols[3:]]
+        return df
 
     def get_company_fundamentals(self, company_ids: List[str]) -> pd.DataFrame:
         """
         :param company_ids: A list of company IDs
         :return: A pandas DataFrame with company fundamental info per company (company_id is a column)
         """
-        return pd.DataFrame.from_records(
-            [ICompanyData.parse_obj(c.dict()).dict() for c in self.get_company_data(company_ids)],
-            exclude=['projected_targets', 'projected_intensities', 'historic_data']).set_index(
-                self.column_config.COMPANY_ID)
+        excluded_cols = ['projected_targets', 'projected_intensities', 'historic_data']
+        df = pd.DataFrame.from_records(
+            [ICompanyData.parse_obj({k:v for k, v in c.dict().items() if k not in excluded_cols}).dict()
+             for c in self.get_company_data(company_ids)]).set_index(self.column_config.COMPANY_ID)
```

Comment:
what does this line do?
looks like one line is missing

Reply:
There's a list comprehension that spans multiple lines.  Perhaps it would be clearer if we put the `.set_index(...)` on the `return df` rather than combining it with the pd.DataFrame constructor?

Reply:
it could be clearer, if we could see, which results each individual step does.
here, i see that there is some parsing of some items in dictionary with excluded columns, converted to dictionary again.

but it seems like an abstract manipulation on a dictionary - it is not clear, which ITR logic it performs, w/o running the code.